src/scripting/lua_bases.cc

```diff
@@ -186,11 +192,11 @@ int LuaEditorGameBase::get_immovable_description(lua_State* L) {
 /* RST
    .. function:: immovable_exists(immovable_name)
 
-      :arg immovable_name: the name of the immovable
+      Returns whether the descriptions know about an ImmovableDescription for the named
+      object. This is either :const:`true` if the named immovable exists, :const:`false` otherwise.
```

Comment:
Linkify ImmovableDescription. 

Grammatical nitpicking: "either" requires an "or", so it should be shortened to "This is true if […], false otherwise"

Oh and the :returns: directive seems to be missing in several places

Reply:
> Oh and the :returns: directive seems to be missing in several places

If it is mentioned in the description i see no reason why we should add it again. If you insist to have *always* the `:returns:` directive i have to rework several places (with the danger of implementing lots of failures :-D ).